commands/features.go

```diff
@@ -0,0 +1,54 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ConradIrwin/font/sfnt"
+)
+
+// Features prints the gpos/gsub tables (contains font features).
+func Features() {
+	if len(os.Args) < 2 {
+		panic(fmt.Errorf("Specify a font file"))
+	}
+
+	file, err := os.Open(os.Args[1])
+	if err != nil {
+		panic(fmt.Errorf("Failed to open font: %s", err))
+	}
+	defer file.Close()
+
+	font, err := sfnt.Parse(file)
+	if err != nil {
+		panic(fmt.Errorf("Failed to parse font: %s", err))
+	}
+
+	layoutTable(font, sfnt.TagGsub, "Glyph Substitution Table (GSUB)")
+	layoutTable(font, sfnt.TagGpos, "Glyph Positioning Table (GPOS)")
+}
+
+func layoutTable(font *sfnt.Font, tag sfnt.Tag, name string) {
+	if font.HasTable(tag) {
+		fmt.Printf("%s:\n", name)
+
+		t := font.Table(tag).(*sfnt.TableLayout)
+		for _, script := range t.Scripts {
+			fmt.Printf("\tScript %q (%s):\n", script.Tag, script.String())
```

Comment:
When I run this on a font I see a weird:

`Script "    " ():`

Do you know why? Can we make it clearer what it means?

Reply:
Would you email me that font?

Reply:
it's here: https://gist.github.com/ConradIrwin/a85f8e432fedf2cd4ea400286a796b5e 

If you have the gist tool installed (brew install gist) you can get the file with: `gist -r a85f8e432fedf2cd4ea400286a796b5e | base64 --decode > adelle.ttf`

Reply:
So that font has a script who's tag is 0x20202020 (four space characters).  The spec says nothing about what this means, so I suspect it is just some weird artefact.

I've changed the tool to not display the () if the name is unknown.